pkgs/by-name/me/metadata/package.nix

```diff
@@ -11,14 +11,17 @@ rustPlatform.buildRustPackage rec {
   pname = "metadata";
   version = "0.1.9";
 
+  # update-crate-ffmpeg-next-version.patch updates Cargo.lock, which doesn't
+  # work well with buildRustPackage.
   src = fetchFromGitHub {
-    owner = "zmwangx";
+    # owner = "zmwangz";
+    owner = "myclevorname";
     repo = "metadata";
-    rev = "v${version}";
-    hash = "sha256-OFWdCV9Msy/mNaSubqoJi4tBiFqL7RuWWQluSnKe4fU=";
+    rev = "a1bc9f53d9aa0aeb17cbb530a1da1de4fdf85328";
```

Comment:
Don't switch the source to another fork. The correct way would be adding https://patch-diff.githubusercontent.com/raw/zmwangx/metadata/pull/11.patch (upload here, instead of fetchpatch, because PRs may be force pushed so they are not stable) to `cargoPatches`

Reply:
Thanks, I didn't know about that attribute.